src/csharp/Extensions/Microsoft.Spark.Extensions.Delta/Microsoft.Spark.Extensions.Delta.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
You don't need to make this change. You can just target `netstandard2.0` and everything will work find on .NET Core 3.0.

The only reason you would multi-target for standard 2.0 and 2.1 is if you wanted to use a new API in 2.1 that wasn't available in 2.0, which it doesn't appear that we are doing.

Reply:
For consideration with targeting `netstandard2.1`:
> .NET Framework won't support .NET Standard 2.1 or later versions. For more details, see the [announcement of .NET Standard 2.1](https://devblogs.microsoft.com/dotnet/announcing-net-standard-2-1/).

Source: https://docs.microsoft.com/en-us/dotnet/standard/net-standard

Reply:
That's why it is targeting both `netstandard2.0` and `netstandard2.1`. .NET Framework will use the `netstandard2.0` compatible version.

Reply:
reverted.